WalletWasabi/Blockchain/TransactionBuilding/BranchAndBound.cs

```diff
@@ -161,17 +168,17 @@ private bool TryFindSolution(long target, [NotNullWhen(true)] out long[]? soluti
 
 	private NextAction GetRandomNextAction()
 	{
-		return _random.Next(0, 2) == 1 ? NextAction.AandB : NextAction.BandA;
+		return _randomFn() ? NextAction.IncludeFirstThenOmit : NextAction.OmitFirstThenInclude;
 	}
 
-	private NextAction GetNextStep(NextAction action)
+	private static NextAction GetNextStep(NextAction action)
```

Comment:
What do we earn by making this static?

Reply:
Nothing really. It was in @adamPetho's PR so I copied it basically not to make the code unnecessarily different.